.github/workflows/daily-aarch64-development.yml

```diff
@@ -0,0 +1,168 @@
+        with:
+          ref: ${{ github.event_name == 'workflow_dispatch' && github.event.ref || 'development' }}
+      - id: generate
+        run: |
+          json=$(.github/generate-native-matrix.sh | tr -d '\n')
+          echo "matrix=${json}" >> $GITHUB_OUTPUT
+
+  aarch64-linux-build-jvm-latest:
+    name: Daily - Aarch64 - Linux - JVM build - Latest Version
+    runs-on: ubuntu-24.04-arm
+    needs: [ generate-json-matrix ]
+    if: "github.repository == 'quarkusio/quarkus-quickstarts' || github.event_name == 'workflow_dispatch'"
+    strategy:
+      fail-fast: false
+      max-parallel: 3
+      matrix: ${{ fromJson(needs.generate-json-matrix.outputs.matrix) }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'workflow_dispatch' && github.event.ref || 'development' }}
+
+      - name: Reclaim Disk Space
+        run: .github/ci-prerequisites.sh
+
+      - name: Get Date
+        id: get-date
+        run: |
+          echo "date=$(/bin/date -u "+%Y-%m")" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: Cache Maven Repository
+        id: cache-maven
+        uses: actions/cache@v4
+        with:
+          path: ~/.m2/repository
+          # refresh cache every month to avoid unlimited growth
+          key: q2maven-jvm-aarch64-${{ steps.get-date.outputs.date }}
+
+      - name: Install JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
+          check-latest: true
+          cache: 'maven'
+
+      - name: Build Quarkus main
+        run: |
+          git clone https://github.com/quarkusio/quarkus.git
+          cd quarkus
+          ./mvnw -T1C -e -B --settings .github/mvn-settings.xml clean install -Dquickly-ci
+
+      - name: Build Quickstarts in JVM mode on aarch64
+        run: |
+          ./mvnw -e -B --settings .github/mvn-settings.xml clean install --fail-at-end -Dstart-containers \
+            -Daarch64 -am -pl "${{ matrix.test-modules }}"
+
+      - name: Zip Artifacts
+        if: failure()
+        run: |
+          zip -R artifacts-jvm-aarch64-${{ matrix.category }}.zip '*-reports/*'
+
+      - name: Archive artifacts
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: artifacts-linux-jvm-aarch64-${{ matrix.category }}
+          path: artifacts-jvm-aarch64-${{ matrix.category }}.zip
+
+      - name: Delete Local Artifacts From Cache
+        shell: bash
+        run: rm -rf ~/.m2/repository/org/acme
+  report-aarch64:
+    name: Report - aarch64
+    runs-on: ubuntu-latest
+    needs: [ aarch64-linux-build-jvm-latest, aarch64-linux-build-native-latest ]
+    if: "always() && github.repository == 'quarkusio/quarkus-quickstarts'"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'workflow_dispatch' && github.event.ref || 'development' }}
+      - uses: quarkusio/report-status-in-issue@main
```

Comment:
Could you also add something similar to what is done here: https://github.com/quarkusio/quarkus/blob/6fc85442437b18594a1c91d1cfdc87c7616b313b/.github/workflows/ci-actions-incremental.yml#L1555-L1574 ?

This way we will have a nice build report in the summary.

You will also have to follow this convention to upload the reports: https://github.com/quarkusio/quarkus/blob/6fc85442437b18594a1c91d1cfdc87c7616b313b/.github/workflows/ci-actions-incremental.yml#L1468-L1484